packages/config/lib/resolve-sets.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const assert = require('assert');
+
+const _ = {
+    uniqWith: require('lodash.uniqwith'),
+    isEqual: require('lodash.isequal')
+};
+
+// TODO: cache
+module.exports = function resolveSets(sets) {
+    return Object.keys(sets).reduce((acc, setName) => {
+        acc[setName] = _.uniqWith(resolveSet(sets[setName], setName, sets), _.isEqual);
+        return acc;
+    }, {});
+}
+
+function resolveSet(setData, setName, sets) {
+    if (typeof setData !== 'string') {
+        return Array.isArray(setData) ? setData : [setData];
+    }
+
+    return setData.split(' ').reduce((setDataAcc, layerStr) => {
+        if (!layerStr.includes('@')) {
+            setDataAcc.push({ layer: layerStr });
+            return setDataAcc;
+        }
+
+        const layerArr = layerStr.split('@');
+        let layerName = layerArr[0];
+        let libName = layerArr[1];
+
+        if (!layerName) {
+            const layerNameArr = libName.split('/');
+            libName = layerNameArr.shift();
+
+            const level = {
+                library: libName
+            };
+
+            if (layerNameArr.length) {
+                level.layer = layerNameArr.join('/');
```

Comment:
layer can contain multiple items joined with slashes ? Do you have an example ?

Reply:
No it shouldnt.

Reply:
cc @tadatuta

Reply:
https://github.com/bem/bem-sdk/pull/213/files#diff-7d30de0af8e04bdd74d45bf96f8df5c4R52